test/test_main.cpp

```diff
@@ -20,14 +20,13 @@
 #define CATCH_CONFIG_RUNNER /* Access to the configuration. */
 #include "tests.hpp"
 
-using namespace Qrack;
+namespace Qrack {
+    enum QInterfaceEngine testEngineType = QINTERFACE_CPU;
```

Comment:
These shouldn’t really be in the Qrack namespace, as they’re consumer-side artifacts.

Reply:
Moved into the global namespace.